### PR TITLE
fix: added filter for all warehouse in the system

### PR DESCRIPTION
**Pull request by vishdha**
TASK ID: [ASANA](https://app.asana.com/0/1199595381635040/1199582410241598)
1. Applied sales and buying cycle warehouse filter to its controller.
2. filtered all warehouses in the system.

### CODE REVIEW

File: erpnext/accounts/doctype/pos_profile/pos_profile.js [hunk 1]
```diff
@@ -68,6 +64,10 @@ frappe.ui.form.on('POS Profile', {
 				}
 			};
 		});
+
+		erpnext.queries.setup_queries(frm, "Warehouse", function() {
+			return erpnext.queries.warehouse(frm.doc);
+		});
```
Comment (hrwX) [inline erpnext/accounts/doctype/pos_profile/pos_profile.js:59]: Merge both the function

**Review by hrwX — CHANGES_REQUESTED** [review]

File: erpnext/accounts/doctype/pos_profile/pos_profile.js (hunk 1 above)
Comment (vishdha) [reply erpnext/accounts/doctype/pos_profile/pos_profile.js:59]: @hrwX  Not understood  what are you trying to say

Comment (hrwX) [reply erpnext/accounts/doctype/pos_profile/pos_profile.js:59]: put the `onload` inside `frappe.ui.form.on`

Comment (vishdha) [reply erpnext/accounts/doctype/pos_profile/pos_profile.js:59]: Refactored onload in pos profile

File: erpnext/erpnext_integrations/doctype/woocommerce_settings/woocommerce_settings.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ frappe.ui.form.on('Woocommerce Settings', {
 	}
 });
 
-frappe.ui.form.on("Woocommerce Settings", "onload", function () {
+frappe.ui.form.on("Woocommerce Settings", "onload", function (frm) {
```
Comment (hrwX) [inline erpnext/erpnext_integrations/doctype/woocommerce_settings/woocommerce_settings.js]: Can you move this onload too ?

**Review by hrwX — CHANGES_REQUESTED** [review]

**Review by hrwX — DISMISSED** [review]

**Review by hrwX — APPROVED** [review]

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**